frontend/app.vue

```diff
@@ -5,4 +5,4 @@
     <NuxtLayout>
         <NuxtPage />
     </NuxtLayout>
-</template>
+</template>
```

Comment:
Somehow many of your files do not end in a newline. In practice this is not really a problem anymore but maybe you can instruct your editor to add a newline at the end of every file.